docs/components/renderer.md

```diff
@@ -26,28 +26,30 @@ The `renderer` component configures a scene's
 | Property                | Description                                                                     | Default Value |
 |-------------------------|---------------------------------------------------------------------------------|---------------|
 | antialias               | Whether to perform antialiasing. If `auto`, antialiasing is disabled on mobile. | auto          |
-| gammaOutput             | Whether to pre-multiply gamma on textures and colors before rendering.          | false         |
+| workflow                | Whether to use color-managed (`linear`) or legacy (`gamma`) workflow.           | gamma         |
```

Comment:
Term workflow seems generic word. Does `colorManagment` make more sense?

Reply:
Or maybe `colorSpace`?

Reply:
To note: prior explanation: 

>  And is "workflow" a term used? Or does it describe "color space"?

> I think either term could be used. C4D has a "Linear Workflow" checkbox. Unity calls it a workflow in their docs, but the checkbox says "Color Space: Linear". Maya requires you to set a bunch of colorspace and color profile settings, all of which together are documented as a "workflow".

> The important thing to know is that this is the colorspace in which renderer calculations are done, and not the color space textures and colors are input as (which is pretty much always sRGB), or the color space output from the renderer. Because that's easy to misunderstand, I think that workflow is a bit clearer than colorSpace.

Reply:
^Per above I think "colorSpace" is too confusing. But `colorManagement: true` might be easier to understand, I do like that...